src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DriverAdapterForFiguresJane.java

```diff
@@ -0,0 +1,35 @@
+//package edu.kis.powp.jobs2d.drivers.adapter;
+//
+//import edu.kis.powp.jobs2d.AbstractDriver;
+//import edu.kis.powp.jobs2d.Job2dDriver;
+//import edu.kis.powp.jobs2d.features.DriverFeature;
+//
+//public class DriverAdapterForFiguresJane extends AbstractDriver {
+
+
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+
+public class DriverAdapterForFiguresJane implements Job2dDriver {
```

Comment:
This class does not solve the problem of FiguresJane patterns.

Reply:
This is not obligatorary. I assume it still does not work. There is no test added with FiguresJane script.